source/overloading.tex

```diff
@@ -2161,7 +2161,7 @@
 that
 \tcode{W}
 did not face
-might be at least as good as
+can be at least as good as
```

Comment:
I don't think "can" really works here. How about:

"Although it is possible that another function `F` that `W` did not face is at least as good as `W`, ..."

Reply:
fixed